### PR TITLE
release 2.11.0

### PR DESCRIPTION
**Pull request by zroubalik**
Relates https://github.com/kedacore/keda/issues/4710

### CODE REVIEW

File: http-add-on/README.md [hunk 1]
```diff
@@ -157,6 +157,11 @@ their default values.
 | `interceptor.tlsHandshakeTimeout` | string | `"10s"` | The maximum amount of time the interceptor will wait for a TLS handshake. Set to zero to indicate no timeout. |
 | `interceptor.tolerations` | list | `[]` | Tolerations for pod scheduling ([docs](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)) |
 
+### Webhooks
```
Comment (tomkerkhove) [inline http-add-on/README.md:160]: Can you revert this please? Probably my local test added this to the http add-on doc template :/

File: keda/Chart.yaml [hunk 2]
```diff
@@ -4,15 +4,15 @@ description: Event-based autoscaler for workloads on Kubernetes
 
 # Specify the Kubernetes version range that we support.
 # We allow pre-release versions for cloud-specific Kubernetes versions such as  v1.21.5-gke.1302 or v1.18.9-eks-d1db3c
-kubeVersion: ">=v1.24.0-0"
+kubeVersion: ">=v1.23.0-0"
```
Comment (tomkerkhove) [inline keda/Chart.yaml:7]: Why revert? We should just remove it imo. No point in enforcing 1.23 if we don't want to enforce things.

**Review by tomkerkhove — APPROVED** [review]

**Review by JorTurFer — APPROVED** [review]

**Comment by tomkerkhove** [comment]
https://github.com/kedacore/charts/discussions/478#discussioncomment-6254672

File: keda/Chart.yaml (hunk 2 above)
Comment (zroubalik) [reply keda/Chart.yaml:7]: well, KEDA won't work on anything below 1.23.0

Comment (zroubalik) [reply keda/Chart.yaml:7]: >= 1.23.0 is a technical reason

>= 1.24.0 or >= 1.25.0 is just a project management reason

🤷‍♂️

Comment (tomkerkhove) [reply keda/Chart.yaml:7]: It can give a wrong impression though but it's fine

**Comment by tomkerkhove** [comment]
> https://github.com/kedacore/charts/discussions/478#discussioncomment-6254672

Please don't forget the notes

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**